frontend/packages/console-app/src/components/admission-webhook-warnings/AdmissionWebhookWarningNotifications.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { AlertVariant } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore-next-line
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  AdmissionWebhookWarning,
+  CoreState,
+  getAdmissionWebhookWarnings,
+  removeAdmissionWebhookWarning,
+} from '@console/dynamic-plugin-sdk/src';
+import { documentationURLs, getDocumentationURL } from '@console/internal/components/utils';
+import { useToast } from '@console/shared/src';
+
+type UseAdmissionWebhookWarnings = () => Map<string, AdmissionWebhookWarning>;
+const useAdmissionWebhookWarnings: UseAdmissionWebhookWarnings = () =>
+  useSelector<CoreState, Map<string, AdmissionWebhookWarning>>(getAdmissionWebhookWarnings);
+
+export const AdmissionWebhookWarningNotifications = () => {
+  const { t } = useTranslation();
+  const toastContext = useToast();
+  const dispatch = useDispatch();
+  const admissionWebhookWarnings = useAdmissionWebhookWarnings();
+  React.useEffect(() => {
+    const docURL = getDocumentationURL(documentationURLs.admissionWebhookWarning);
```

Comment:
Just hard code the kube doc url on admission webhooks here? This isn't an OKD/OpenShift specific concept, it's a part of kube.

Reply:
Please don't hard code it in this file, but rather put it in https://github.com/openshift/console/blob/6c050bebfd222e4d97ef15b9148258936261d0c9/frontend/public/components/utils/documentation.tsx#L13-L25